### ISSUE TITLE
a bug of padding

### ISSUE THREAD

**Issue by Frank-Jin54** [pytorch/pytorch#27246]
## 🐛 Bug
conv.py line 339 340 self.padding is float and throw an exception

change to  return F.conv2d(input, weight, self.bias, self.stride,
                        tuple([int(i) for i in self.padding]), self.dilation, self.groups)

## To Reproduce

1.
1.
1.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by SsnL** [comment]
This only happens if you pass a float to `nn.Conv2d` constructor, which you should not do.

**Comment by Frank-Jin54** [comment]
Oh, yes, thanks for quick reply.
